plugins/nginx-vhosts/commands

```diff
@@ -76,6 +77,7 @@ EOF
       fi
 
       NOSSL_SERVER_NAME=$(echo $NONSSL_VHOSTS | tr '\n' ' ')
+      docker cp $DOKKU_APP_CONTAINER_ID:/app/nginx.conf $DOKKU_ROOT/$APP/nginx.conf.template 2> /dev/null || true
```

Comment:
Also, does this fail if an nginx.conf.template file already exists?

Should we also support the ssl template file?